### PR TITLE
Add main menu

### PR DESCRIPTION
**Pull request by ZoazVA**

### CODE REVIEW

**Comment by AWarmCupOfTea** [comment]
Let's follow https://cbea.ms/git-commit/ for naming our PRs and commits.

File: scenes/Menu.gd [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# var a = 2
+# var b = "text"
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	pass # Replace with function body.
+
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
```
Comment (AWarmCupOfTea) [inline scenes/Menu.gd:14]: Let's remove the unnecessary comments.

**Review by AWarmCupOfTea — CHANGES_REQUESTED** [review]

File: scenes/Menu.gd (hunk 1 above)
Comment (ZoazVA) [reply scenes/Menu.gd:14]: Ok I'll do it now

**Comment by AWarmCupOfTea** [comment]
And please check the article I linked above. I would rename the commit to something like "Add main menu"

File: scenes/Menu.gd [hunk 2]
```diff
@@ -0,0 +1,11 @@
+extends Control
+func _ready():
```
Comment (AWarmCupOfTea) [inline scenes/Menu.gd:2]: Please add spacing between the functions like described in https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html

**Review by AWarmCupOfTea — CHANGES_REQUESTED** [review]

**Review by AWarmCupOfTea — APPROVED** [review]

**Pull request merged**